Formula/h/helidon.rb

```diff
@@ -0,0 +1,23 @@
+class Helidon < Formula
+  desc "Command-line tool for Helidon application development"
+  homepage "https://helidon.io/"
+  url "https://github.com/helidon-io/helidon-build-tools/archive/refs/tags/3.0.6.tar.gz"
+  sha256 "749cf3fd162bb9449ab57584c0bdf8874114d678499071ea522c047637de0f90"
+  license "Apache-2.0"
+
+  depends_on "maven"
```

Comment:
Does it need this at runtime?

Reply:
Yes it does, if not set it will throw an exception during test.
See https://github.com/Homebrew/homebrew-core/actions/runs/6122041216/job/16617069297
Reason is, the generated project by Helidon CLI can be run through a `dev loop`, building project with maven when user makes changes to the generated code.